utils/generate-commands-json.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+# Generate a commands.json file from the output of the `COMMAND` command
+# Prerequisites: Python 3.5+ and redis-py
+import argparse
+import functools
+import json
+import redis
+
+def convert_list_to_dict(arr, recurse):
+    rep = {}
+    for i in range(0,len(arr),2):
+        key = arr[i].replace('-', '_')
+        val = arr[i+1]
+        if type(val) is list and recurse:
+            val = convert_list_to_dict(val, recurse)
+        rep[key] = val
+    return rep
+
+def convert_flags_to_truthy_dict(flags):
+    rep = {}
+    for f in flags:
+        rep[f] = True
+    return rep
+
+def convert_argument(arg):
+    rep = convert_list_to_dict(arg, False)
+    if 'flags' in rep:
+        rep = {
```

Comment:
doesn't this override `rep` with a dict containing just the flags?

Reply:
oh ok i see.. please use
```
rep.update(convert_flags_to_truthy_dict(rep['flags']))
```
other places too

Reply:
or even better
```
rep.update(convert_flags_to_truthy_dict(rep.pop('flags', [])))
```
and you can drop `if 'flags' in rep`

Reply:
Yep, I forgot all my Python